app/Http/Controllers/Three/SignupsController.php

```diff
@@ -43,7 +44,6 @@ public function __construct(SignupService $signups)
         $this->transformer = new SignupTransformer;
 
         $this->middleware('auth:api', ['only' => ['store', 'update', 'destroy']]);
-        $this->middleware('role:admin', ['only' => ['store', 'update', 'destroy']]); // @TODO: Allow anyone to use this.
```

Comment:
I think we still want `role:admin` on the destroy endpoint!

Reply:
@DFurnes added back in in [this PR](https://github.com/DoSomething/rogue/pull/591)!